aclocal.m4

```diff
@@ -1,6 +1,6 @@
-# generated automatically by aclocal 1.16.5 -*- Autoconf -*-
+# generated automatically by aclocal 1.13.4 -*- Autoconf -*-
```

Comment:
looks like this was generated with an older version.  Please regenerate.

Reply:
Thanks - I have regenerated.